class/practice7.1/src/MaybeTheBestCodeEver.cpp

```diff
@@ -0,0 +1,35 @@
+#include <iostream>
+#include <vector>
+#include <print>
+#include <Maybe.hpp>
+
+int main() {
+    RectCollection collection;
+
+    for (int i = 1; i <= 5; i++) {
```

Comment:
5 should be a named constant for example constexpr as you are using it in multiple places(line 9, line 28 in this file)